backend/src/packages/files/file.service.ts

```diff
@@ -70,7 +70,7 @@ class FileService implements Service {
       }),
     );
 
-    return { url: presignedUrl };
+    return { url: presignedUrl, contentType: payload.contentType };
```

Comment:
We decided to update our approach and let's remove presignedUrl and everything related to it, getPreSignedUrl etc and return entity from create method as we usually do. So that we have the same link in db and returned link

Reply:
Done